src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,264 @@
+package duke;
+
+
+import duke.cmds.Cmds;
+import duke.deadline.Deadline;
+import duke.duke_exception.DukeException;
+import duke.event.Event;
+import duke.event.ToDo;
+import duke.task.Task;
+import netscape.javascript.JSObject;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.JSONValue;
+
+public class Duke {
+    private static final String MARK_CMD = "mark";
+    private static final String UNMARK_CMD = "unmark";
+    private static final String LIST_CMD = "list";
+    private static final String ADD_CMD = "add";
+    private static final String DELETE_CMD = "delete";
+    private static final String OUT_DIR = "output";    // Honestly create a config file or smth
```

Comment:
Use comments on separate line with same indentation as code, so that the logical structure of code is not broken.